### PR TITLE
Update helm templates to configure and apply deamonset for nodes only

### PR DESCRIPTION
**Pull request by redhatrises**
- This enhancement is allows for future changes to add additional components like the installing both a container sensor and node sensor.

### CODE REVIEW

File: helm-charts/falcon-sensor/values.yaml [hunk 1]
```diff
@@ -2,44 +2,48 @@
...
+    labels: {}
+
+    updateStrategy: RollingUpdate
+
+    # Various pod security context settings. Bear in mind that many of these have an impact
+    # on the Falcon Sensor working correctly.
+    #
+    # - User that the container will execute as. Typically necessary to run as root (0).
+    # - Runs the Falcon Sensor containers as privileged containers. Typically not
+    #   necessarily unless running within environments such as OpenShift.
```
Comment (shawndwells) [inline helm-charts/falcon-sensor/values.yaml:23]: Can we point to documentation, or provide high-level guidance, on when privileged containers would be needed (or not)?

Eg does "such as OpenShift" also include Rancher and Pivotal/VMWare? Where do we send users to learn the answer?

File: helm-charts/falcon-sensor/values.yaml [hunk 2]
```diff
@@ -2,44 +2,48 @@
...
+    securityContext:
+      runAsUser: 0
+      privileged: true
+      readOnlyRootFilesystem: false
+      allowPrivilegeEscalation: true
+
+  image:
+    repository: falcon-sensor
```
Comment (shawndwells) [inline helm-charts/falcon-sensor/values.yaml:31]: Should the images be renamed to ``sensor-node-container`` to match earlier references in the daemonsets.yaml file?

File: helm-charts/falcon-sensor/values.yaml (hunk 1 above)
Comment (redhatrises) [reply helm-charts/falcon-sensor/values.yaml]: The Falcon Linux sensor (which should be used for nodes) has to be privileged regardless of kubernetes environments.
Updated to state this.

File: helm-charts/falcon-sensor/values.yaml (hunk 2 above)
Comment (redhatrises) [reply helm-charts/falcon-sensor/values.yaml]: Updated them both to be falcon-node-sensor

**Review by shawndwells — APPROVED** [review]

**Pull request merged**